### PR TITLE
fix: [PL-29925]: fix error msg if connector not found in org and account scope

### PR DESCRIPTION
**Pull request by bhavya181**
## Describe your changes
fix error msg if connector not found in org and account scope

tested locally by finding account level connector that doesn't exists
Screenshot 2023-04-03 at 7 40 11 PM

## Checklist
- [x] I've documented the changes in the PR description.
- [x] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- 
  Specific builds
  - `trigger manager`
  - `trigger dms`
  - `trigger ng_manager`
  - `trigger cvng `
  - `trigger cmdlib`
  - `trigger template_svc`
  - `trigger events_fmwrk_monitor`
  - `trigger event_server`
  - `trigger change_data_capture`
  -  Trigger multiple builds together: For eg: `trigger dms, manager`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>

You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- Trigger CommonChecks: `trigger commonchecks`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger utAll`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
- Go Build: `trigger gobuild`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/46017)

### CODE REVIEW

**Comment by bhavya181** [comment]
trigger messagecheck

File: 440-connector-nextgen/src/main/java/io/harness/connector/apis/resource/ConnectorResource.java [hunk 1]
```diff
@@ -171,9 +171,18 @@ public ConnectorResource(@Named("connectorDecoratorService") ConnectorService co
       @BeanParam GitEntityFindInfoDTO gitEntityBasicInfo) {
     Optional<ConnectorResponseDTO> connectorResponseDTO =
         connectorService.get(accountIdentifier, orgIdentifier, projectIdentifier, connectorIdentifier);
+    String connectorNotFountErrorMsg =
+        String.format("Connector with identifier [%s] not found in project [%s] and org [%s].", connectorIdentifier,
+            projectIdentifier, orgIdentifier);
+    if (null == orgIdentifier) {
```
Comment (nishantsaini20) [inline 440-connector-nextgen/src/main/java/io/harness/connector/apis/resource/ConnectorResource.java:177]: Add UTs covering these cases.

**Review by nishantsaini20 — CHANGES_REQUESTED** [review]

**Review by ankit-harness — APPROVED** [review]

**Review by ankit-harness — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/46017#-:-NS9G-X861bswD0n-fo7:bnfp4nl)* status: 1 file(s) to review. 1 change requests. 1 of 2 approvals obtained.

**Review by bhavya181 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/46017)* status: 2 file(s) to review. 1 change requests. 1. unresolved discussions. 0 of 2 approvals obtained. 

___
*[`440-connector-nextgen/src/main/java/io/harness/connector/apis/resource/ConnectorResource.java` line 177 at r1](https://reviewable.io/reviews/harness/harness-core/46017#-NS6RQsfcuZ5goR7mE3y-r1-177:-NSARIvgD6EXsuXYoWBw:bb7f2mf) ([raw file](https://github.com/harness/harness-core/blob/a104143cc5b62d1ead8ef52fdc4db11d1133c147/440-connector-nextgen/src/main/java/io/harness/connector/apis/resource/ConnectorResource.java#L177)):*
<details><summary><i>Previously, nishantsaini20 (Nishant Saini) wrote…</i></summary><blockquote>

Add UTs covering these cases.
</blockquote></details>

done

**Review by nishantsaini20 — APPROVED** [review]

**Pull request merged**